07.ls_object/long_formatter.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative 'file_detail'
+
+class LongFormatter
+  def initialize(names)
+    @names = names
+  end
+
+  def format
```

Comment:
通常、formatは整形してその結果を戻り値として返すときに使うメソッド名ですね。

```ruby
formatted_text = format_text(text)
puts formatted_text
```

このメソッドはputsしていて戻り値は使われないので、たとえば

```suggestion
  def print_data
```

みたいな名前の方が実態に即してそうです。

また、理想を言えば、formatメソッドにしたまま、putsせずに文字列として戻り値を返せるのがベストです。
https://qiita.com/jnchito/items/e2b1c8904cd7b8e3ce50